user-guide/Cloud_Platform/AboutCloudPlatform/DataMiner_Cloud_Packs.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: CloudPackages
+---
+
+# DataMiner Cloud Packs
+
+DataMiner Cloud Packs are packages that include the necessary DataMiner Extension Modules to enable DataMiner Cloud Platform capabilities.
```

Comment:
DataMiner Cloud packs are installation packages that include the necessary DataMiner Extension Modules which enable DataMiner Cloud Services ?